gpMgmt/bin/gpexpand

```diff
@@ -776,7 +776,7 @@ class SegmentTemplate:
         localHostname = getLocalHostname()
         # NOTE: Currently, GP's scp transfers relative file paths into the home
         # directory by default.
-        pathsAreSame = (os.path.realpath(self.segTarFile) == os.path.join('/home/%s/' % getUserName(), self.schema_tar_file))
+        pathsAreSame = os.path.realpath(self.segTarFile) == os.path.join('/home/', getUserName(), self.schema_tar_file)
```

Comment:
os.path.join already joins by '/', so explicit is not needed

Reply:
I prefer writing directories with a trailing slash. Paths will be normalized anyway, so who cares?

Reply:
under windows `os.path.join` joins paths by `\` for example

Reply:
In Windows, most of the time using normal slash is also valid. 
We use Linux here? And `/home` is an absolute path beginning with a slash. I don't see a problem still :C